sdk/storage/perf-test/src/main/java/com/azure/storage/blob/perf/UploadBlobTest.java

```diff
@@ -21,6 +27,6 @@ public void run() {
 
     @Override
     public Mono<Void> runAsync() {
-        return blobAsyncClient.upload(RandomFlux.create(options.getSize()), null, true).then();
+        return blobAsyncClient.upload(randomByteBufferFlux, null, true).then();
```

Comment:
Should be better for perf since we don't create new flux every iteration.

Reply:
Though is it safe to reuse the same flux for each iteration?  Or is the flux "used up" after the first upload so we would need to create a new one every time?

Reply:
Yeah, tested it by using the same flux to read multiple times. Same data was returned each time.

Reply:
Would it also work with a static flux, if multiple instances were reading in parallel?  Or does it need to be an instance flux, so it can be reused across iterations, but each instance needs its own copy?  Shouldn't matter much for perf either way, but I like to scope variables correctly to increase understanding of the minimal test requirements.

Reply:
It could have been static if the since of the buffer was fixed. Since the size is determined by the options provided in the constructor, I have made this an instance flux. However, as we discussed, we could make this static and initialize in `globalSetup()` as the options are not going to change for a run and all instances share the same option. I didn't want to make that assumption here as it didn't matter to the performance of the test itself.